resources/views/dashboard/admin/events/new.blade.php

```diff
@@ -10,7 +10,7 @@
 <div class="container">
     {{ html()->form()->route('saveNewEvent')->acceptsFiles()->open() }}
         @csrf
-        <div class="form-group">
+        <div class="form-group mb-3">
```

Comment:
A thought, as for code cleanliness.

Since we are using SASS, can we take advantage of `@apply` to avoid duplicating this everywhere? It seems clear that we want this on most (if not all) `form-group`s, and taking advantage of sass's mixins could help with code duplication.